### PR TITLE
Out of gas during return / revert

### PR DESCRIPTION
**Pull request by qbzzt**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -27,14 +27,13 @@ sani-vm: $(vm_tests:=.format) $(vm_fillers:=.format) \
          $(vm_tests:=.filled)
 
 # TODO: enable $(gs_fillers:=.valid) $(gs_tests:=.format) $(gs_fillers:=.format)
-sani-gs: $(gs_tests:=.valid) \
```
Comment (winsvega) [inline Makefile:30]: I thought this is changed in develop. did you do the rebase?

File: src/GeneralStateTestsFiller/stCreateTest/CreateResultsFiller.yml [hunk 2]
```diff
@@ -0,0 +1,229 @@
+CreateResults:
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stCreateTest/CreateResultsFiller.yml:1]: why is here a create result test?

File: src/GeneralStateTestsFiller/stMemExpandingEIP150Calls/OOGinReturnFiller.yml [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+          ; Other constants
+          (def 'NOP 0)   ; No OPeration
+
+          ; Understand the input.
+          [type]       $4
+          [gas2Use]    $36
+
+          ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+          [0] 0x60A760A7
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stMemExpandingEIP150Calls/OOGinReturnFiller.yml:60]: did you try with empty memory?

File: src/GeneralStateTestsFiller/stCreateTest/CreateResultsFiller.yml (hunk 2 above)
Comment (winsvega) [reply src/GeneralStateTestsFiller/stCreateTest/CreateResultsFiller.yml:1]: needs rebase

**Review by winsvega — CHANGES_REQUESTED** [review]
needs rebase due to the conflict with other prs

**Comment by qbzzt** [comment]
Will replace with a clear PR

**Pull request closed without merging**